crates/matrix-sdk/Cargo.toml

```diff
@@ -35,13 +35,16 @@ rustls-tls = ["reqwest/rustls-tls"]
 socks = ["reqwest/socks"]
 sso_login = ["warp", "rand", "tokio-stream"]
 appservice = ["ruma/appservice-api-s", "ruma/appservice-api-helper", "ruma/rand"]
+image_proc = ["image"]
+image_rayon = ["image/jpeg_rayon"]
 
 docs = [
     "encryption",
     "sled_cryptostore",
     "sled_state_store",
     "sso_login",
-    "qrcode"
+    "qrcode",
+    "image_proc"
```

Comment:
Maybe add a trailing comma to simplify future diffs?

Reply:
Done